test/livebook/live_markdown/markdown_helpers_test.exs

```diff
@@ -253,6 +377,18 @@ defmodule Livebook.LiveMarkdown.MarkdownHelpersTest do
       assert markdown == reformat(markdown)
     end
 
+    test "raw html at beginning of line" do
+      markdown = "mypara\n<hr />"
+
+      assert markdown == reformat(markdown)
+    end
+
+    test "raw html not at beginning of line" do
+      markdown = "mypara\n  <hr />"
+
+      assert markdown == reformat(markdown)
+    end
```

Comment:
These two examples are from the "Rendering of block and inline elements" section of the `earmark-parser` README. The first one fails because of whitespace changes.

Reply:
The whitespace just gets normalized (which is the case for all other blocks) and that's expected.

Reply:
In the first case, the formatter adds an extra line before the `hr` (likely correct) but it also formats the `hr` to `<hr  />` (that's TWO spaces before the `/`)